docs/dev/build.md

```diff
@@ -344,56 +401,217 @@ There are two options how to use OpenVINO on Apple Silicon:
 - (Native) Compile OpenVINO for arm64 architecture with extra module [OpenVINO ARM plugin](https://github.com/openvinotoolkit/openvino_contrib/tree/master/modules/arm_plugin) location in [OpenVINO Contrib](https://github.com/openvinotoolkit/openvino_contrib). Note, build steps will cover this as a default scenario.
 - (Rosetta) Compile Intel CPU plugin `x86_64` architecture and run under [Rosetta].
 
-1. Clone contrib repo:
+The software was validated on:
+- macOS 11.x, 12.x, arm64
+
+1. (Get sources) Clone submodules:
 ```sh
-cd ..
-git clone https://github.com/openvinotoolkit/openvino_contrib.git
-cd openvino_contrib
-git submodule update --init --recursive
-cd ../openvino
+% git clone https://github.com/openvinotoolkit/openvino.git
```

Comment:
We already cloned all repositories... We can just mention it here that we need to have both repositories...

Reply:
Done in https://github.com/openvinotoolkit/openvino/pull/16063/commits/562c6c32912eb936d9ca384c7caa9cf772ce2fa2